### PR TITLE
Added optional sdk param to pick timeout value

### PR DESCRIPTION
**Pull request by arnasbr**
Related to https://app.zenhub.com/workspaces/devops-60184ef23de345001a78baee/issues/gh/traveltime-dev/traveltime-python-sdk/106

### CODE REVIEW

File: traveltimepy/sdk.py [hunk 1]
```diff
@@ -81,6 +81,10 @@
 from geojson_pydantic import FeatureCollection
 
 
+def default_timeout() -> int:
```
[the remark this answers is not in the document]
Comment (danielnaumau) [reply traveltimepy/sdk.py:84]: and have you checked, was default timeout equal to 1800?

**Review by danielnaumau — CHANGES_REQUESTED** [review]

File: traveltimepy/sdk.py [hunk 2]
```diff
@@ -172,25 +181,32 @@ async def geocoding_async(
...
             self._sdk_params,
+            timeout,
             self._geocoding_reverse_params(lat, lng),
         )
 
     async def supported_locations_async(
-        self, locations: List[Location]
+        self,
+        locations: List[Location],
+        timeout: int = default_timeout(),
```
[the remark this answers is not in the document]
Comment (arnasbr) [reply traveltimepy/sdk.py:201]: reworking it to be in sdk params anyways

File: traveltimepy/sdk.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (arnasbr) [reply traveltimepy/sdk.py:84]: default was 300 (5min)

File: traveltimepy/sdk.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (arnasbr) [reply traveltimepy/sdk.py:201]: that's probably what you meant, but yes, will have a fix soon

**Pull request merged**